### PR TITLE
Use ansible-pulp repo (renamed)

### PR DESCRIPTION
**Pull request by asmacdo**
re #4508

### CODE REVIEW

**Review by dkliban — APPROVED** [review]

File: playbooks/source-install.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@
     - name: Set up Vagrant machine for source installs
       include: ../vagrant/vagrant-pretask.yml
   roles:
-    - pulp3-postgresql
+    - pulp3-database
     - pulp3-workers
     - pulp3-resource-manager
     - pulp3-webserver
```
Comment (mdellweg) [inline playbooks/source-install.yml:16]: Should those be renamed to `pulp-*`, too?

**Review by mdellweg — COMMENTED** [review]
Is in the forklift update anything worth mentioning?

File: playbooks/source-install.yml (hunk 1 above)
Comment (asmacdo) [reply playbooks/source-install.yml:16]: Yup that's next. Just didn't want to leave it broken for too long.

**Comment by asmacdo** [comment]
Nice catch @mdellweg, the forklift change was accidental.

**Pull request merged**